pkg/constants/constants.go

```diff
@@ -319,6 +319,10 @@ const (
 	// HiveFeatureGatesEnabledEnvVar is the the environment variable specifying the comma separated list of
 	// feature gates that are enabled.
 	HiveFeatureGatesEnabledEnvVar = "HIVE_FEATURE_GATES_ENABLED"
+
+	// AWSPrivateLinkControllerConfigFileEnvVar if present, points to a simple text
+	// file that includes configuration for aws-private-link-controller
+	AWSPrivateLinkControllerConfigFileEnvVar = "AWS_PRIVATELINK_CONTROLLER_CONFIG_FILE"
```

Comment:
Hmm. AFAIK we've never done a config file for a controller. I'm fine with it, but want to make sure that this is a pattern we now want to adopt for other controllers as well.

@dgoodwin thoughts?  ^^

Reply:
Yes definitely outside of this change.